doajtest/unit/test_index_searchbox.py

```diff
@@ -45,19 +45,19 @@ def test_04_legacy_search_routes(self):
                 # A plain /search goes to the journal search
                 resp = t.get('/search')
                 assert resp.status_code == 301, resp.status_code
-                assert resp.location == url_for('doaj.journals_search', _external=True), resp.location
+                assert resp.location == url_for('doaj.journals_search'), resp.location
```

Comment:
seem like behaviour of resp.location changed 
it will return short url in flask 2.0